.github/actions/lint_code/action.yaml

```diff
@@ -23,3 +23,14 @@ runs:
     - name: 'Run shellcheck'
       shell: bash
       run: ./utils/scripts/shellcheck.sh
+    - name: Install node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 20
+    - name: 'Run nodejs lint'
+      shell: bash
+      working-directory: ./utils/build/docker/nodejs/express4
+      run: |
+        npm install
```

Comment:
What about installing only lint deps to  speed up the job ?

Reply:
done!

Reply:
I see you added `-D`. Did you test it to make sure it was only installing the dev deps ? I just tried it and I don't think it's doing that ?

Reply:
There seem to have been different ways: `--dev`, `--only=dev`, `--production=false` and `-D` and I've read comments stating that `-D` was working with the latest npm versions

But I haven't checked it :S

Reply:
There may not be a way to install only the dev dependencies...

Reply:
Back to doing `npm install eslint eslint-standard....` then i guess

Reply:
what about using the global arg ? `-g` that shouldn't use any package/lock file no ?

Reply:
I don't know if it could work with the other dependencies, `eslint` yes, but `eslint-plugin-mocha`,... I think that they should be in `node_modules`.
I mentioned to Igor to try with something like:
```shell
cp package.json package-back.json && node -pe "const pack = require('./package.json');delete pack.dependencies;require('fs').writeFileSync('./package.json',JSON.stringify(pack))" && npm install --no-package-lock axios && mv package-back.json package.json
```

It should install only `devDependencies`, because we are removing `dependencies` from `package.json`.
Just to see how much time can we earn, if the time difference is insignificant... we can just maintain the normal `npm install`

Reply:
Around the same time:
- npm install -> 41s
- your hack -> 43s